mlx/traceability.py

```diff
@@ -180,6 +183,12 @@ def process_item_nodes(app, doctree, fromdocname):
         node['line'] = node.line
         node.perform_replacement(app, env.traceability_collection)
 
+    for item_id, item_info in ChecklistItemDirective.query_results.copy().items():
+        if match(r"[A-Z\d_]+-[A-Z\d_]+", item_id):
```

Comment:
add numbers here as well and I think `-`also needs to be included?

Reply:
`\d` matches numbers. Right now only one `-` is allowed (and mandatory) right after the slug. Is that too restrictive?

Reply:
Yes, I think mandatory `-` is a bit too restrictive. So I would propose `r"[A-Z\d_-]+"`, but indeed this could be something we can have configurable in `conf.py`

Reply:
done: d24c7ef1cba27f38cfe5c00d551ddefe2d5a27ca